### PR TITLE
Setup experience improvements

### PR DESCRIPTION
**Pull request by yourordinarycat**
**Resolved / Related Issues**
None.

**Details of Changes**
These changes improve the code used during setup, fixing a memory leak, adding lottie animations, improving performance, improving the way localized strings are handled, and removing unused code.

**Validation**
- [x] Built and ran the app
- [x] Compared to the previous setup
- [x] Went through the setup

**Screenshots / Videos (optional)**
https://imgur.com/a/7tBN9Gq

### CODE REVIEW

**Comment by itsWindows11** [comment]
Is this still WIP?

**Comment by yourordinarycat** [comment]
There's a few improvements I want to make before requesting a review, mostly to the navigation related code.

File: Rise Media Player Dev/Dialogs/SetupDialogContent.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+namespace Rise.App.Dialogs
+{
+    public sealed partial class SetupDialogContent : Page
+    {
+        private SettingsViewModel ViewModel => App.SViewModel;
+
+        public SetupDialogContent()
+        {
+            InitializeComponent();
+            Navigate(SlideNavigationTransitionEffect.FromBottom);
```
Comment (itsWindows11) [inline Rise Media Player Dev/Dialogs/SetupDialogContent.xaml.cs]: Keep the FromRight transition maybe?

File: Rise Media Player Dev/Dialogs/SetupDialogContent.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+    {
+        private void BackButton_Click(object sender, RoutedEventArgs e)
+        {
+            ViewModel.SetupProgress--;
+            Navigate(SlideNavigationTransitionEffect.FromLeft);
+        }
+
+        private async void PrimaryButton_Click(object sender, RoutedEventArgs e)
+        {
+            switch (ViewModel.SetupProgress)
```
Comment (itsWindows11) [inline Rise Media Player Dev/Dialogs/SetupDialogContent.xaml.cs]: We can use an if statement for this instead.

**Review by itsWindows11 — CHANGES_REQUESTED** [review]
Good changes, but I think you should make it look more like the old dialog i.e. leave more space for the animated icon and make it larger.

Other changes to the code are suggested below.

**Comment by yourordinarycat** [comment]
> Good changes, but I think you should make it look more like the old dialog i.e. leave more space for the animated icon and make it larger.

The visual player is technically the same size (128x128), but the animation itself seems to be a bit smaller. It looks close with a -16 margin though:

image

**Comment by itsWindows11** [comment]
I mean make the right side smaller like the old setup dialog.

**Comment by itsWindows11** [comment]
Like in here for example:
image

**Comment by yourordinarycat** [comment]
Should be good now:

image

**Review by itsWindows11 — APPROVED** [review]
LGTM

**Pull request merged**